cli/command/registry/logout.go

```diff
@@ -42,7 +42,7 @@ func runLogout(dockerCli command.Cli, serverAddress string) error {
 		loggedIn        bool
 		regsToLogout    []string
 		hostnameAddress = serverAddress
-		regsToTry       = []string{serverAddress}
+		regsToTry       []string
```

Comment:
Looks like this means that in the `isDefaultRegistry` situation we won't do any logging out, correct?

Reply:
You're right. Thanks for pointing this out! I handled the default registry case in my new commits.